webapp/components/_new/features/SearchResults/SearchResults.story.js

```diff
@@ -0,0 +1,148 @@
+import { storiesOf } from '@storybook/vue'
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-18T08:15:39Z" title="Friday, September 18th 2020, 10:15:39 am +02:00">Sep 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,148 @@
+import { storiesOf } from '@storybook/vue'
```


Unfortunately `yarn storybook` is not working at my machine at the moment.

What’s about you?